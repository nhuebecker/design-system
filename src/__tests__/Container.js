import React from 'react'
import renderer from 'react-test-renderer'
import { Container } from '..'

describe('Container', () => {
  test('renders with default theme max-width', () => {
    const json = renderer.create(<Container />).toJSON()
    expect(json).toMatchSnapshot()
  })

  test('renders with maxWidth', () => {
    const json = renderer.create(<Container maxWidth={32} />).toJSON()
    expect(json).toMatchSnapshot()
    expect(json).toHaveStyleRule('max-width', '32rem')
  })
})
