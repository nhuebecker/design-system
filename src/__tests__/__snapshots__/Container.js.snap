// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container renders with default theme max-width 1`] = `
.c1 {
  margin-left: auto;
  margin-right: auto;
}

.c0 {
  max-width: 72rem;
}

<div
  className="c0 c1"
/>
`;

exports[`Container renders with maxWidth 1`] = `
.c1 {
  margin-left: auto;
  margin-right: auto;
}

.c0 {
  max-width: 32rem;
}

<div
  className="c0 c1"
/>
`;
